src/hotspot/os/linux/os_linux.cpp

```diff
@@ -2138,6 +2166,10 @@ void os::Linux::print_process_memory_info(outputStream* st) {
   }
 #endif // __GLIBC__
 
+#ifdef __GLIBC__
```

Comment:
You can merge these two `#ifdef __GLIBC__` blocks.

Reply:
Will do.